src/SearchResults.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import Moment from "react-moment";
+import "moment-timezone";
+
+const SearchResults = prop => {
+  const [isSelected, setSelect] = useState();
```

Comment:
Bitting right into the lesson two stuff here :) There is one bit  you haven't done, the Highlight booking row when clicked thingy, right now it highlights all rows instead of just the one that was clicked, beacause the state is managed here rather than in the rows